functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -86,15 +95,15 @@ def scylla_services_names(db_cluster: ScyllaPodCluster) -> list:
             if name not in ('NAME', f"{scylla_cluster_name}-client")]
 
 
-def wait_for_resource_absence(db_cluster: ScyllaPodCluster,
+def wait_for_resource_absence(db_cluster: ScyllaPodCluster,  # pylint: disable=too-many-arguments
                               resource_type: str, resource_name: str,
+                              namespace: str = SCYLLA_NAMESPACE,
                               step: int = 2, timeout: int = 60) -> None:
     def resource_is_absent() -> bool:
         all_resources = db_cluster.k8s_cluster.kubectl(
-            f"get {resource_type} -o=custom-columns=:.metadata.name",
-            namespace=SCYLLA_NAMESPACE,
+            f"get {resource_type} -o=custom-columns=:.metadata.name", namespace=namespace,
         ).stdout.split()
-        return resource_name not in all_resources
+        return not all_resources or resource_name not in all_resources
```

Comment:
I'm little bit puzzled by this addition: `.split()` always returns a list and `anything not in []` is always True